src/einsteinpy/tests/test_symbolic/test_ricci.py

```diff
@@ -34,3 +34,24 @@ def test_TypeError():
         assert False
     except TypeError:
         assert True
+
+
+def test_Ricci_Scalar():
+    symbolstr = "r theta phi t"
+    syms = sympy.symbols(symbolstr)
+    list2d = (np.zeros(shape=(4, 4), dtype=int)).tolist()
+    list2d[0][0] = 1
+    list2d[1][1] = syms[0] ** 2
+    list2d[2][2] = (syms[0] ** 2) * (sympy.sin(syms[1]) ** 2)
+    list2d[3][3] = syms[0] ** 2
+    metric = MetricTensor(list2d, syms)
+    T = RicciTensor.from_metric(metric)
+    G, M, c, a = sympy.symbols("G M c a")
+    list2d = np.zeros((4, 4), dtype=int).tolist()
+    list2d[0][0] = 1 - (a / syms[1])
+    list2d[1][1] = -1 / ((1 - (a / syms[1])) * (c ** 2))
+    list2d[2][2] = -1 * (syms[1] ** 2) / (c ** 2)
+    list2d[3][3] = -1 * (syms[1] ** 2) * (sympy.sin(syms[2]) ** 2) / (c ** 2)
+    sch = MetricTensor(list2d, syms)
+    scalar = RicciScalar(sch, T)
```

Comment:
Why are two separate metrics used here? The RicciScalar and RicciTensor should both be constructed from the same metric. Furthermore, in the second construction of ``list2d``, you seem to be building a Schwarzschild metric, but you have here what looks like
![equation](https://latex.codecogs.com/gif.latex?\begin{bmatrix}&space;1-\frac{a}{\theta}&space;&&space;0&space;&&space;0&space;&&space;0\\&space;&&space;\frac{-1}{c^2(1-\frac{a}{\theta})}&space;&&space;&&space;\\&space;0&space;&&space;0&space;&&space;-\frac{\theta^2}{c^2}&space;&&space;0\\&space;0&space;&&space;0&space;&&space;0&space;&&space;-\frac{\theta^2}{c^2}\sin(\phi)^2&space;\end{bmatrix})

Reply:
I thought that this was already proven what I wanted to proof